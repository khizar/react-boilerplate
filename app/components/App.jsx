import React from 'react';
import CSSModules from 'react-css-modules';
import styles from './App.pcss';

const App = props => <div styleName="main">hellooo</div>;

export default CSSModules(App, styles, { allowMultiple: true });
