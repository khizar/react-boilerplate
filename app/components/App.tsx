import React from 'react';
import { hot } from 'react-hot-loader/root';
import CSSModules from 'react-css-modules';
import Hello from './TypedApp';

import styles from './App.pcss';

const App = () => (
  <div styleName='main'>
    <Hello compiler='ts' framework='react' />
  </div>
);

export default hot(CSSModules(App, styles, { allowMultiple: true }));
