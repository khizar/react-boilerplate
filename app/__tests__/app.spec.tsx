import React from 'react';
import { render } from '@testing-library/react';

import App from '../components/App';

describe('App component is mounted', () => {
  it('should match snapshot', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
