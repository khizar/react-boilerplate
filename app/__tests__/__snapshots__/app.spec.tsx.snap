// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component is mounted should match snapshot 1`] = `
<div>
  <div
    class="main"
  >
    <h1>
      Hello from 
      ts
       and 
      react
      !
    </h1>
  </div>
</div>
`;
